src/providers/wms/qgswmscapabilities.cpp

```diff
@@ -1569,6 +1569,20 @@ void QgsWmsCapabilities::parseWMTSContents( const QDomElement &element )
 
         style.legendURLs << legendURL;
       }
+      QDomElement thirdChildElement = secondChildElement.firstChildElement( QStringLiteral( "LegendURL" ) );
+      if ( !thirdChildElement.isNull() )
+      {
+        QgsWmtsLegendURL legendURL;
+
+        legendURL.format   = thirdChildElement.attribute( QStringLiteral( "format" ) );
+        legendURL.minScale = thirdChildElement.attribute( QStringLiteral( "minScale" ) ).toDouble();
```

Comment:
Shouldn't this be minScaleDenominator and maxScaleDenominator?

From http://schemas.opengis.net/wmts/1.0/wmtsGetCapabilities_response.xsd:

```
<element name="LegendURL">
<annotation>
<documentation>
Zero or more LegendURL elements may be provided, providing an image(s) of a legend relevant to each Style of a Layer. The Format element indicates the MIME type of the legend. minScaleDenominator and maxScaleDenominator attributes may be provided to indicate to the client which scale(s) (inclusive) the legend image is appropriate for. (If provided, these values must exactly match the scale denominators of available TileMatrixes.) width and height attributes may be provided to assist client applications in laying out space to display the legend.
</documentation>
</annotation>
<complexType>
<complexContent>
<extension base="ows:OnlineResourceType">
<annotation>
<documentation>
The URL from which the legend image can be retrieved
</documentation>
</annotation>
<attribute name="format" type="ows:MimeType">
<annotation>
<documentation>A supported output format for the legend image</documentation>
</annotation>
</attribute>
<attribute name="minScaleDenominator" type="double">
<annotation>
<documentation>
Denominator of the minimum scale (inclusive) for which this legend image is valid
</documentation>
</annotation>
</attribute>
<attribute name="maxScaleDenominator" type="double">
<annotation>
<documentation>
Denominator of the maximum scale (exclusive) for which this legend image is valid
</documentation>
</annotation>
</attribute>
```